pennylane_lightning/core/src/simulators/lightning_qubit/gates/cpu_kernels/GateImplementationsPI.hpp

```diff
@@ -1,1394 +0,0 @@
-// Copyright 2018-2023 Xanadu Quantum Technologies Inc.
-
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-
-//     http://www.apache.org/licenses/LICENSE-2.0
-
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-/**
- * @file
- * Defines gate operations with precomputed indices
- */
-#pragma once
-
-/// @cond DEV
-// Required for compilation with MSVC
-#ifndef _USE_MATH_DEFINES
-#define _USE_MATH_DEFINES // for C++
-#endif
-/// @endcond
-
-#include <bit>
-#include <complex>
-#include <vector>
-
-#include "BitUtil.hpp"
-#include "GateIndices.hpp"
```

Comment:
Do we want to keep `GateIndices.hpp`?

Reply:
Why should we remove it? :thinking:

Reply:
I thought  only `GateImplementationsPI.hpp` requires `GateIndices.hpp`. Never mind.